cross/fish/PLIST

```diff
@@ -0,0 +1,15 @@
+bin:bin/fish
+bin:bin/fish_indent
+bin:bin/fish_key_reader
+rsc:etc/fish/
```

Comment:
I suppose that custom configurations are made in the files located in the `etc/fish` folder.
On a package update/upgrade this folder will be lost, so I propose to move these files to `${SYNOPKG_PKGDEST}/var/`.
So you must add a custom post installation to move the files to the new location and need to configure fish-shell to use this specific folder.
 
You can test whether the `Package upgrade completed successfully`  when you manually install the same package again (without prior uninstall). If your custom settings are restored then you managed it.

Reply:
There's several other projects that include `rsc:etc` in their cross/project/PLIST files.
For the upgrade scenario, I followed rsnapshot ([source](https://github.com/SynoCommunity/spksrc/blob/b883d5616639685198522ba48fdc672c9982c881/spk/rsnapshot/src/service-setup.sh#L9-L19)) and others as examples.
I did an install, edited /usr/local/fish/etc/* files on the system, changed the SPK_REV, upgraded, and fish/etc was successfully backed up and restored and working as expected on the system, using the restored etc files.

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully